traffic_router/connector/src/main/java/com/comcast/cdn/traffic_control/traffic_router/secure/CertificateRegistry.java

```diff
@@ -17,59 +17,108 @@
 
 public class CertificateRegistry {
-	private static final Logger log = Logger.getLogger(CertificateRegistry.class);
-	private CertificateDataConverter certificateDataConverter = new CertificateDataConverter();
-	volatile private Map<String, HandshakeData>	handshakeDataMap = new HashMap<>();
-	private RouterNioEndpoint sslEndpoint = null;
-	final private Map<String, CertificateData> previousData = new HashMap<>();
-
-	// Recommended Singleton Pattern implementation
-	// https://community.oracle.com/docs/DOC-918906
-	private CertificateRegistry() {
-	}
-
-	public static CertificateRegistry getInstance() {
-		return CertificateRegistryHolder.DELIVERY_SERVICE_CERTIFICATES;
-	}
-
-	public List<String> getAliases() {
-		return new ArrayList<>(handshakeDataMap.keySet());
-	}
-
-	public HandshakeData getHandshakeData(final String alias) {
-		return handshakeDataMap.get(alias);
-	}
-
-	public Map<String, HandshakeData> getHandshakeData() {
-	    return handshakeDataMap;
+    private static final Logger log = Logger.getLogger(CertificateRegistry.class);
+    private CertificateDataConverter certificateDataConverter = new CertificateDataConverter();
+    volatile private Map<String, HandshakeData>	handshakeDataMap = new HashMap<>();
+    private RouterNioEndpoint sslEndpoint = null;
+    final private Map<String, CertificateData> previousData = new HashMap<>();
+    public static final String DEFAULT_ALIAS = "_default_";
+
+    // Recommended Singleton Pattern implementation
+    // https://community.oracle.com/docs/DOC-918906
+    private CertificateRegistry() {
     }
 
-	public void setEndPoint(final RouterNioEndpoint routerNioEndpoint) {
-		sslEndpoint = routerNioEndpoint;
-	}
+    public static CertificateRegistry getInstance() {
+        return CertificateRegistryHolder.DELIVERY_SERVICE_CERTIFICATES;
+    }
+
+    public List<String> getAliases() {
+        return new ArrayList<>(handshakeDataMap.keySet());
+    }
 
-	@SuppressWarnings("PMD.AccessorClassGeneration")
-	private static class CertificateRegistryHolder {
-		private static final CertificateRegistry DELIVERY_SERVICE_CERTIFICATES = new CertificateRegistry();
-	}
+    public HandshakeData getHandshakeData(final String alias) {
+        return handshakeDataMap.get(alias);
+    }
 
-	@SuppressWarnings({"PMD.CyclomaticComplexity", "PMD.AvoidDeeplyNestedIfStmts", "PMD.NPathComplexity"})
-	synchronized public void importCertificateDataList(final List<CertificateData> certificateDataList) {
-		final Map<String, HandshakeData> changes = new HashMap<>();
-		final Map<String, HandshakeData> master = new HashMap<>();
+    public Map<String, HandshakeData> getHandshakeData() {
+        return handshakeDataMap;
+    }
 
-		// find CertificateData which has changed
-		for (final CertificateData certificateData : certificateDataList) {
-			try {
-				final String alias = certificateData.alias();
+    private void addToHandshakeData(final String key, final HandshakeData handshakeData) {
```

Comment:
nit: this one-line method seems unnecessary. Can the calling code could just call the one line in this method instead?

Reply:
no longer included with new updates